docs/ECLStandardLibraryReference/SLR-Mods/EncodeBase64.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE sect1 PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN"
+"http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd">
+<sect1 id="EncodeBase64">
+  <title>EncodeBase64</title>
+
+  <para><emphasis role="bold">STD.Str.EncodeBase64<indexterm>
+      <primary>STD.Str.EncodeBase64</primary>
+    </indexterm><indexterm>
+      <primary>Str.EncodeBase64</primary>
+    </indexterm><indexterm>
+      <primary>EncodeBase64</primary>
+    </indexterm>(</emphasis> <emphasis>value</emphasis> <emphasis
+  role="bold">)</emphasis> <emphasis role="bold"></emphasis></para>
+
+  <informaltable colsep="1" frame="all" rowsep="1">
+    <tgroup cols="2">
+      <colspec colwidth="80.50pt" />
+
+      <colspec />
+
+      <tbody>
+        <row>
+          <entry><emphasis>value</emphasis></entry>
+
+          <entry>A DATA value containing the data to encode.</entry>
+        </row>
+
+        <row>
+          <entry>Return:<emphasis> </emphasis></entry>
+
+          <entry>EncodeBase64 returns a STRING value.</entry>
+        </row>
+      </tbody>
+    </tgroup>
+  </informaltable>
+
+  <para>The <emphasis role="bold">EncodeBase64 </emphasis>function returns a
+  STRING containing the binary data encoded in Base64. </para>
+
+  <para>Example:</para>
+
+  <programlisting format="linespecific">IMPORT STD;
+str:=D'HeWxUZCOqwVoZVkCZIwhAWtnGDB';
```

Comment:
It would make the example clearer if this variable were not called str (since if contains data) and if it contained something meaningful rather than something that looks like it has already been encoded.

Ideally your input to the encode function should be something like X'0102030405' and your input to the decode function should be whatever that encodes to ('AQIDBAU=')